### ISSUE TITLE
Call of Duty: Modern Warfare 3 Italian version does not work

### ISSUE THREAD

**Issue by mardy** [Fledge68/WiiFlow_Lite#369]
**Describe the bug**
Hi! The issue is that this game does not work under WiiFlow, but properly works under USB Loader GX ([v3.0-r1281](https://github.com/wiidev/usbloadergx/releases/tag/v3.0-r1281), I didn't try other versions): it shows the "Loading..." text forever, exactly how it's shown in this [old video](https://youtu.be/zm-om_oQtSs?si=W70EWoHMJW0XKg97&t=22).

The interesting part is that the NTSC version of the same game works fine (well, after setting cIOS 250) -- I didn't try other European versions, but if the Italian version has this problem it's possible that other PAL versions also don't load.

**To Reproduce**

**What version of WiiFlow Lite**
v5.5.3

**On Wii or Wii U vWii**
Wii

**Additional context**
I tried also v5.5.4 with cIOS set to AUTO: it sent me back to the Wii system screen, and after that point WiiFlow wouldn't see the USB drive anymore (rebooting, changing USB ports, nothing would work -- reverting back to v5.5.3 fixed this). I'll report this as a separate issue.

**Comment by Fledge68** [comment]
lets stick to v.5.5.3 for now.

is your wii NTSC or PAL?

for your PAL game did you try changing the game IOS from auto to any other cIOS?

did you change any game video mode settings?

**Comment by Fledge68** [comment]
sorry i have no idea what the problem would be. but if the ntsc version works why not just use it?

**Comment by mardy** [comment]
Yes, sure, not a big issue, I just thought that since it works with USB Loader GX maybe there was some trick that you could take from there -- just in case there are more games who have similar issues.

**Comment by Fledge68** [comment]
@wiidev any thoughts? strange this happens on v5.5.3

**Comment by wiidev** [comment]
@mardy If the system language isn't set to Italian then go into the games settings and set the language to Italian.

If that doesn't work then it'd be good to have [SysCheck report](https://wii.guide/syscheck.html) and test [WiiFlow Lite v5.4.4](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.4) to see if an older release worked.

**Comment by wiidev** [comment]
@mardy If the system language isn't set to Italian then go into the games settings and set the language to Italian.

If that doesn't work then it'd be good to have a [SysCheck report](https://wii.guide/syscheck.html) and test [WiiFlow Lite v5.4.4](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.4) to see if an older release works.

**Comment by mardy** [comment]
So, after trying @wiidev's suggestions, it looks like this is a regression from v5.4.4. Here's what I tried:
- installed v 5.4.4, set the game language to italian, started it on cIOS 249 -> Stuck on "Loading..."
- started the game with cIOS 250 -> **it worked!**
- then went back to 5.5.3, set the game language to italian, started it on cIOS 249 -> Stuck on "Loading..."
- started the game with cIOS 250 -> Wii stuck (the loading bar from WiiFlow does not even appear)
- tried all other availables cIOS -> Wii stuck (the loading bar from WiiFlow does not even appear)

Here's my [SysCheck.csv](https://github.com/Fledge68/WiiFlow_Lite/files/12794450/SysCheck.csv)

**Comment by wiidev** [comment]
With the games cIOS configured to 250 and the language set to Italian (if necessary?) could you try upgrading the loader until it stops working? So [v5.4.6](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.6), [v5.4.7](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.7), [v5.4.9](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.9) etc.

With that information we could try to narrow down which version introduced the issue.

**Comment by mardy** [comment]
5.4.4 is the last working version. Here are other versions that I tried:
- 5.5.0: Core dump
- 5.4.8: Wiiflow shows its loading animation (as if wiiflow rebooted), then hangs
- 5.4.6: same as 5.4.8
- 5.4.5: the cIOS is stuck to AUTO, so starting the game gets stuck at the "Loading..." text

By the way, the language switch does not matter, since indeed my Wii's language is set to Italian.

**Comment by wiidev** [comment]
> * 5.4.5: the cIOS is stuck to AUTO, so starting the game gets stuck at the "Loading..." text

v5.4.6 was released to address that issue, which is why I skipped v5.4.5. But v5.4.7 and v5.4.9 include a lot of big changes.

I originally suggested v5.4.4 because it's the version that I installed recently so that I could compare the video mode settings against v5.4.5 and newer.

You've got all of your video mode settings set to their defaults, right? So the games video mode is "default" and the loaders video mode is set to "game"?

**Comment by mardy** [comment]
> You've got all of your video mode settings set to their defaults, right? So the games video mode is "default" and the loaders video mode is set to "game"?

Yes, before installing any version I first removed the old `wiiflow` directory, so they all had the default settings.

**Comment by wiidev** [comment]
Could you try with [v5.4.9](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.4.9), [v5.5.1](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.5.1) and [v5.5.2](https://github.com/Fledge68/WiiFlow_Lite/releases/tag/v5.5.2)? One of these might solve that WiiFlow hang/crash and the game might also work.

**Comment by Fledge68** [comment]
my 2 cents. call of duty games require a cios with base 57 which is why the ntsc version works when he sets it to 250 (a base 57 cios).  why the pal version doesn't work with cios 250 is the problem.  there must be something pal related.

**Comment by Fledge68** [comment]
so after reviewing all your comments in this issue.

it appears to me that after you downgraded to v5.4.4 whenever you upgraded to a higher version, especially v.5.5.3, you did not upgrade the ext_booter.bin (in the bins folder) which explains why higher versions all either black screened, no scroll bar, only wiiflow loading animation, crashed, or returned to wii system menu.

I think wiidev is right. you need to set the game's language to 'italian' and ios to '250' and then launch the game. i read where you did that and it didn't work but im betting that's because you didn't upgrade ext_booter.bin.

also it probably works in usblgx because wiidev added the 'auto' launguge setting which auto changes the language to italian.
